lib/Conversation/CommentList.js

```diff
@@ -5,13 +5,13 @@ const CommentList = ({ comments, ...rest }) => {
   const list = comments.map(comment =>
```

Comment:
The `comments` property on this component has no defaultProp set; maybe it will be okay anyway because an empty array is perhaps being sent from the parent, but for linting's sake it's probably worth adding a `defaultProps` to comments on this component.